src/gameplay/cards/MonsterCard.js

```diff
@@ -77,23 +77,54 @@ class MonsterCard extends Card {
   }
 
   takeSnapshot() {
+    let summonAction;
+
+    if (this.lv < 5) {
+      summonAction = {
+        name: 'summon',
+        desc: 'summon this monster to the field',
+        in: Game.HAND,
+        params: [
+          {select: Game.SLOT, in: Game.MONSTER_SLOTS, of: Game.SELF, desc: 'select an empty slot in the monster slots to summon'},
+          {select: Game.DISPLAY, desc: 'select the display of the monster'},
+          {select: Game.POSE, desc: 'select the pose of the monster'},
+        ],
+      };
+    } else if (this.lv < 7) {
+      summonAction = {
```

Comment:
See above fix.

Reply:
Done.

Created `createAction` and `createActionParam` functions to create actions and its params.